src/classes/STG_InstallScript.cls

```diff
@@ -146,8 +146,8 @@ global without sharing class STG_InstallScript implements InstallHandler {
             mappings.add(new Affl_Mappings__c(Name = 'Business Organization', Account_Record_Type__c = 'Business Organization', Primary_Affl_Field__c = 'Primary Business Organization'));
```

Comment:
@ceiroa @skosgi1 On Samsheeth's demo call to Ryan and I today, we brought up the idea that it might be worthwhile for new customers to automatically see the API name for the Primary Affiliation Field instead of the label -- that way new customers are already supported in multiple languages.

Should we create a followup WI for this? It /sounds/ like a simple change, but I think we want to get UX/PM involved in that decision since it would change the UI for new customers. We might want to update the settings UI to show the friendly name of the label even if the API name is stored in the settings, so it could be something we think about during the EDA Settings rewrite.

Reply:
Interesting idea. I agree that we'd want UX/PM involved. Opening an item for a future investigation into this sounds good to me.

Reply:
I can create a work item in the backlog to look at this. I think the best approach is to show the label, but when they click EDIT it shows the API Name. We would have to document this however and explain how to get at the API Name for those that don't know how to find it or even know what it is.
Or, we have to get much more involved with how those are setup and make it more "wizard" like.

Reply:
I've created W-037866 to look into this.

Reply:
Thanks, Ryan!

Reply:
Thanks Ryan and Bobby!